CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Bugs Fixed
 
 - [[#2152]](https://github.com/Azure/azure-sdk-for-c/pull/2152) Fix value in user agent string.
+- [[#2162]](https://github.com/Azure/azure-sdk-for-c/pull/2162) Remove $version requirement in IoT Hub and IoT Edge reported properties response topic.
```

Comment:
One idea would be to add "Twin" reported properties to distinguish from telemetry ones.

Reply:
thanks for the feedback cris!

Reply:
I don't want it to sound like $version will no longer be in the IoT Hub topic, and I'm concerned the current tag line could be interpreted to mean that...

Reply:
Please see update.